### PR TITLE
build(bazel): use ng_package entry_point_name for umd bundles

### PR DESCRIPTION
**Pull request by jelbourn**
This change also requires updating the package gold test to work with multiple test packages.

### CODE REVIEW

**Review by alexeagle — COMMENTED** [review]
should there also be added files for the custom-entry-point-name?

**Comment by jelbourn** [comment]
Ah, they didn't get `git add`-ed. For some reason WebStorm's git integration does not like angular/angular

**Review by alexeagle — APPROVED** [review]

File: packages/bazel/test/ng_package/example-custom-entry-point-name.golden [hunk 1]
```diff
@@ -0,0 +1,939 @@
+bundles
```
[the remark this answers is not in the document]
Comment (jelbourn) [reply packages/bazel/test/ng_package/example-custom-entry-point-name.golden]: Removed second golden and changed `entry_point_name` on the example package. The unit tests for core/common already make assertions that capture the default behavior.

File: packages/bazel/test/ng_package/example_package.golden [hunk 2]
```diff
@@ -729,15 +730,15 @@ MyModule.ctorParameters = () => [];
  * Generated bundle index. Do not edit.
  */
 
-export { MyModule };
-//# sourceMappingURL=example.js.map
+export { SecondaryModule, a };
```
[the remark this answers is not in the document]
Comment (jelbourn) [reply packages/bazel/test/ng_package/example_package.golden]: That's odd- I'm not sure my change would have caused this

**Pull request closed without merging**